docs/en/observability/slo-create.asciidoc

```diff
@@ -0,0 +1,123 @@
+
+From here, complete the following:
+
+* <<define-sli,Define your service-level indicator (SLI)>>
+* <<set-slo>>
+* <<slo-describe>>
+
+[discrete]
+[[define-sli]]
+== Define your SLI
+
+The type of SLI to use depends on the location of your data. If you're creating an SLO based on raw logs coming from your services, you can use a custom KQL SLI. If you're creating an SLO based on services using application performance monitoring (APM), you can use an APM latency or APM availability SLI. See the following table for more on each type of SLI:
+
+[cols="1,1"]
+|===
+
+|*Custom KQL* 
+|This indicator can be based on any elasticsearch index or index pattern you have. You define two queries, one that yields the good events and one that yields the total events from your index. 
+
+*Example:* You could define a custom KQL indicator based on the `service-logs` with the good query defined as `nested.field.response.latency <= 100 and nested.field.env : “production”` and the total query defined as `nested.field.env : “production”`. 
+|*APM latency* 
+|This indicator is based on the APM data that we receive from your instrumented services and a latency threshold. 
+
+*Example:* You could define an indicator on an APM service named `banking-service` for the `production` environment, and the transaction name `POST /deposit` with a latency threshold value of 300ms.
+|*APM availability* 
+|This indicator is based on the APM data that we receive from your instrumented services and a list of good status codes. 
+
+*Example:* You could define an indicator on an APM service named `search-service` for the `prod` environment, and the transaction name `POST /search` with the good status codes set to `[200, 300, 400]`.
+|===
+
+[discrete]
+[[custom-kql-sli]]
+=== Custom KQL
+When defining a custom KQL SLI, you can set the following fields:
+
+* *Index* — The index or index pattern you want to base the SLI upon.
+* *Timestamp field* — The timestamp field used by the index.
+* *Query filter* — A filter to apply on the index.
+* *Good query* — The KQL query yielding the good events to take into account for computing the SLI.
+* *Total query* — The KQL query yielding all events to take into account for computing the SLI.
+
+[discrete]
+[[apm-latency-sli]]
+
+=== APM latency or availability
+When defining an APM latency or APM availability SLI, you can set the following fields:
+
+* *Service name* — The APM service name.
+* *Service environment* — Either `all` or the specific environment.
+* *Transaction type* — Either `all` or the specific transaction type.
+* *Transaction name* — Either `all` or the specific transaction name.
+* *Threshold (APM latency only)* — The latency threshold in milliseconds (ms) to consider the request as good.
+* *Good status codes (APM availability only)* — The list of status codes used to differentiate the good and bad requests. Default: [2xx, 3xx, 4xx].
```

Comment:
This is in a code snippet earlier in the page.

```suggestion
* *Good status codes (APM availability only)* — The list of status codes used to differentiate the good and bad requests. Default: `[2xx, 3xx, 4xx]`.
```

Reply:
We recently removed this field